### PR TITLE
Feature/services docs

### PR DESCRIPTION
**Pull request by jyecusch**

### CODE REVIEW

File: docs/services-templates.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+The function template repository is available on [GitHub](https://github.com/nitrictech/function-templates) and contains the official Nitric Function-as-a-Service templates, for creating Serverless Functions in a variety of languages.
+
+The templates available in this repository at the time of writing are:
+
+| Language   | Runtime     | Identifier        |
+| ---------- | ----------- | ----------------- |
+| TypeScript | Node.js 12  | function/nodets12 |
+| JavaScript | Node.js 12  | function/nodejs12 |
+| Python 3   | CPython 3.7 | function/python37 |
```
Comment (davemooreuws) [inline docs/services-templates.mdx:31]: CPython 3.7 typo

File: docs/services-templates.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+| Java       | OpenJDK 11  | function/java11   |
+| Go         | Go 1.5      | function/java11   |
```
Comment (davemooreuws) [inline docs/services-templates.mdx:33]: java11 should be golang15?

**Review by davemooreuws — CHANGES_REQUESTED** [review]
Looks good, just a few changes.

File: docs/services-templates.mdx (hunk 1 above)
Comment (tjholm) [reply docs/services-templates.mdx:31]: CPython is correct, thats the default python interpreter, will include additional runtimes in future as part of other templates though.

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**